app/assets/javascripts/controllers/container_live_dashboard/container_live_dashboard_controller.js

```diff
@@ -115,8 +115,42 @@ miqHttpInject(angular.module('containerLiveDashboard', ['ui.bootstrap', 'pattern
       $scope.refresh();
     };
 
+    var doRefreshTenant = function (action) {
```

Comment:
Any chance you could do the initialization only in one place?

Like, do it only in this function and call it from the controller body?

Because at some point, somebody will fix one of the urls but not the other.. :)